### ISSUE TITLE
Staging ECS tasks are failing to load

### ISSUE THREAD

**Issue by maurizi** [PublicMapping/districtbuilder#689]
## Description
Since at least April 1st our ECS tasks for the staging cluster have been unable to load during the allowed startup period, and have been repeatedly failing health checks and cycling out.

Based on ECS task logs it appears that our `health_check_grace_period_seconds` terraform variable is not being respected and/or we are experiencing out-of-memory issues

## Logs
We noticed a few errors like the following in the ECS task logs for the cycling tasks as well (not sure if this is related or irrelevant):
```
[Nest] 28 - 04/12/2021, 8:57:59 PM [TopologyService] Unable to download static files from for s3://global-districtbuilder-dev-us-east-1/regions/US/WA/2020-10-22T14:34:20.488Z/: socket hang up +12622ms
```

**Comment by maurizi** [comment]
After manually updating the health check timeouts associated with the staging load balancer to the maximum allowed, the ECS tasks began failing with an out-of-memory failure instead:
```
2021-04-13 12:25:35
<--- Last few GCs --->
2021-04-13 12:25:35
[29:0x49abe30] 721721 ms: Scavenge 17699.9 (17802.7) -> 17697.9 (17816.2) MB, 74.3 / 0.0 ms (average mu = 0.673, current mu = 0.738) allocation failure
2021-04-13 12:25:35
[29:0x49abe30] 721945 ms: Scavenge 17713.9 (17816.2) -> 17715.8 (17818.2) MB, 75.3 / 0.0 ms (average mu = 0.673, current mu = 0.738) allocation failure
2021-04-13 12:25:35
FATAL ERROR: Scavenger: semi-space copy Allocation failed - JavaScript heap out of memory
2021-04-13 12:25:35
<--- Last few GCs --->
2021-04-13 12:25:35
[29:0x49abe30] 721721 ms: Scavenge 17699.9 (17802.7) -> 17697.9 (17816.2) MB, 74.3 / 0.0 ms (average mu = 0.673, current mu = 0.738) allocation failure
2021-04-13 12:25:35
[29:0x49abe30] 721945 ms: Scavenge 17713.9 (17816.2) -> 17715.8 (17818.2) MB, 75.3 / 0.0 ms (average mu = 0.673, current mu = 0.738) allocation failure
2021-04-13 12:25:35
[29:0x49abe30] 722700 ms: Scavenge 17715.9 (17818.2) -> 17715.1 (17823.2) MB, 754.4 / 0.0 ms (average mu = 0.673, current mu = 0.738) allocation failure
2021-04-13 12:25:35
<--- JS stacktrace --->
2021-04-13 12:25:35
Cannot get stack trace in GC.
2021-04-13 12:25:35
<--- Last few GCs --->
2021-04-13 12:25:35
[29:0x49abe30] 721721 ms: Scavenge 17699.9 (17802.7) -> 17697.9 (17816.2) MB, 74.3 / 0.0 ms (average mu = 0.673, current mu = 0.738) allocation failure
2021-04-13 12:25:35
[29:0x49abe30] 721945 ms: Scavenge 17713.9 (17816.2) -> 17715.8 (17818.2) MB, 75.3 / 0.0 ms (average mu = 0.673, current mu = 0.738) allocation failure
2021-04-13 12:25:35
[29:0x49abe30] 722700 ms: Scavenge 17715.9 (17818.2) -> 17715.1 (17823.2) MB, 754.4 / 0.0 ms (average mu = 0.673, current mu = 0.738) allocation failure
2021-04-13 12:25:35
FATAL ERROR: Scavenger: semi-space copy Allocation failed - JavaScript heap out of memory
2021-04-13 12:25:35
<--- JS stacktrace --->
2021-04-13 12:25:35
Cannot get stack trace in GC.
2021-04-13 12:25:35
<--- Last few GCs --->
2021-04-13 12:25:35
[29:0x49abe30] 721721 ms: Scavenge 17699.9 (17802.7) -> 17697.9 (17816.2) MB, 74.3 / 0.0 ms (average mu = 0.673, current mu = 0.738) allocation failure
2021-04-13 12:25:35
[29:0x49abe30] 721945 ms: Scavenge 17713.9 (17816.2) -> 17715.8 (17818.2) MB, 75.3 / 0.0 ms (average mu = 0.673, current mu = 0.738) allocation failure
2021-04-13 12:25:35
[29:0x49abe30] 722700 ms: Scavenge 17715.9 (17818.2) -> 17715.1 (17823.2) MB, 754.4 / 0.0 ms (average mu = 0.673, current mu = 0.738) allocation failure
2021-04-13 12:25:35
FATAL ERROR: Scavenger: semi-space copy Allocation failed - JavaScript heap out of memory
2021-04-13 12:25:35
<--- JS stacktrace --->
2021-04-13 12:25:35
Cannot get stack trace in GC.
```

**Comment by maurizi** [comment]
Closed by #690
